.eslintrc.js

```diff
@@ -11,8 +11,10 @@ module.exports = {
     ecmaVersion: 2018,
     sourceType: 'module',
     project: './tsconfig.eslint.json',
+    extraFileExtensions: ['.json'],
   },
   rules: {
     '@typescript-eslint/explicit-module-boundary-types': 'off',
+    '@typescript-eslint/no-unused-vars': ['error', { varsIgnorePattern: '_' }],
```

Comment:
Since we're configuring it as an `error`, shouldn't it be better to use [`noUnusedLocals`](https://www.typescriptlang.org/tsconfig#noUnusedLocals) and [`noUnusedParameters`](https://www.typescriptlang.org/tsconfig#noUnusedParameters) in `tsconfig.json` instead? I'm just asking because as I generally prefer to use TS flags on my personal projects, I'd love to see a second opinion on this :sweat_smile:

Reply:
I like the eslint option because we can ignore `_` variables.
I'm not sure if this is also the case with the TS flags.

Reply:
Hmm yes, AFAIK we can't ignore underscored **variables** with them, only parameters.